apps/projects/app/utils/shapes.js

```diff
@@ -30,9 +30,30 @@ const issueShape = PropTypes.shape({
 }).isRequired
 
 const userGitHubShape = PropTypes.shape({
-  avatarUrl: PropTypes.string.isRequired,
+  avatarUrl: PropTypes.string,
```

Comment:
It seems removing `isRequired` will hide potential warning messages, because at least `PanelComponents.js`->`Avatar` component is using this `prop` without any explicit check for its value, so it is actually a required prop unless we add any kind of check to that component (maybe there are other components involved as well)